lib/index.js

```diff
@@ -1,63 +1,56 @@
-const RSS = require("rss")
-const url = require("url")
+const RSS = require('rss')
 
 module.exports = (options = {}) => {
   const { feedOptions, limit, encoding, destination, collection } = {
     feedOptions: {},
     limit: 20,
-    encoding: "utf8",
-    destination: "rss.xml",
-    collection: "posts",
-    ...options,
+    encoding: 'utf8',
+    destination: 'rss.xml',
+    collection: 'posts',
+    ...options
   }
 
-  return function(files, metalsmith, done) {
+  return function (files, metalsmith, done) {
     if (!feedOptions.site_url) {
-      return done(new Error("feedOptions.site_url must be configured"))
+      return done(new Error('feedOptions.site_url must be configured'))
     }
 
     if (feedOptions.feed_url == null) {
-      feedOptions.feed_url = url.resolve(feedOptions.site_url, destination)
+      feedOptions.feed_url = new URL(destination, feedOptions.site_url)
```

Comment:
self note: check this is properly working, the tests are way to light to be trusted…